addon/mixins/application-route-mixin.js

```diff
@@ -60,11 +60,11 @@ export default Ember.Mixin.create({
   /**
     This action is triggered whenever the session is successfully
     authenticated. If there is a transition that was previously intercepted
-    by
-    [`AuthenticatedRouteMixin#beforeModel`](#SimpleAuth-AuthenticatedRouteMixin-beforeModel)
+    by 
+    {{#crossLink "Mixins.AuthenticatedRouteMixin/beforeModel:method"}}AuthenticatedRouteMixin#beforeModel{{/crossLink}}
```

Comment:
I'd say the `@namespace`s should be removed - this would of course require e.g. the `Base` authenticator to be renamed to `BaseAuthenticator` so it doesn't conflict with e.g. the `Base` authorizer (which would have to be renamed to `BaseAuthorizer`) etc.

Another solution would be to keep the namespaces but then there should be matching modules that export a namespace with all the correct members. Otherwise I think that will be confusing.